tests/test_app.py

```diff
@@ -342,6 +342,14 @@ def test_purge_plugins():
     assert 'destroyme' not in app.plugins
 
 
+def test_create_parser():
+    from sampleproject.toolkit.app import BaseApplication
+
+    app = BaseApplication()
+    parser = app.create_parser(add_help=False)
+    parser.add_argument("--help", action="help")
```

Comment:
Perhaps add a couple of assertions?

Reply:
Le fait d'appeler parser.add_argument valide qu'on a bien un parser créé.

Le fait d'ajouter `--help` valide que `add_help=False` est bien transmis.

À quelle assertion penses-tu ?

Reply:
> qqch qui valide ce que fait create_parser(), par exemple le
> positionnement de 'prog'.

Adopté. Merci pour l'idée.